content/javascript/concepts/arrays/terms/toSorted/toSorted.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: '.toSorted()'
+Description: 'Takes an array and returns a new array with all the elements sorted in ascending order.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Comparison'
+  - 'Functions'
+  - 'Methods'
+  - 'Sort'
+  - 'Unicode'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toSorted()`** method takes an array and returns a new array with the elements sorted in ascending order. It does not mutate the original array. All undefined elements are sorted to the end of the array.
+
+It does this by converting array elements to [strings](https://www.codecademy.com/resources/docs/javascript/strings) and comparing them by [Unicode code point value](https://developer.mozilla.org/en-US/docs/Glossary/Code_point). 
+
+## Syntax
+
+**Functionless**
+```pseudo
+array.toSorted();
+```
+
+**Arrow function**
+'a' is the first element for comparison and 'b' is the second.
+
+```pseudo
+array.toSorted((a, b) => { /* Code for a compare function. */ });
+```
+
+**Compare function**
+'compareFn' specifies a function that defines the sort order.
+
+```pseudo
+function compareFn() {
+  /* Code for a compare function. */
+};
+
+array.toSorted(compareFn);
+```
+
+**Inline compare function**
+```pseudo
+array.toSorted(function compareFn(a, b) { /* … */ });
+``` 
+
+## Example
+
+In the below example, there are two arrays - `numbers`, containing integers & floats (a number with a decimal place), and `sports`, containing strings.
+
+```js
+const sports = ["baseball", "basketball", "tennis", "pickleball", undefined, ""]
+const numbers = [-3, 4, 1, 9, 3, 2, 1, 200, 2159.20, 959, undefined, ""];
+
+let sportsSorted = sports.toSorted();
+let numbersSorted = numbers.toSorted();
+
+console.log('Sports: ' + sportsSorted);
+console.log('Numbers: ' + numbersSorted);
+```
```

Comment:
add an empty line

Reply:
✅ in [5a59c46](https://github.com/Codecademy/docs/pull/2716/commits/5a59c46fe45eb19a8b49792b070baee48386a5a4).